src/server/service/index.ts

```diff
@@ -11,3 +12,10 @@ export const driveV3 = google.drive({
     scopes: ["https://www.googleapis.com/auth/drive"],
   }),
 });
+
+mailchimp.setConfig({
+  apiKey: env.MAILCHIMP_API_KEY,
+  server: "us22",
```

Comment:
How did you determine `us22`?

Reply:
I believe this is statically determined by mailchimp per account and "the last digits" of the API key.

Reply:
Ah ok. Should we extract the server based of `MAILCHIMP_API_KEY`? Maybe in `server.schema` create a new key called `MAILCHIMP_SERVER`?

Reply:
what's considered "the last digits"? Can we always determine the server prefix based on the API key?

Reply:
"if the URL is https://us6.mailchimp.com/account/api/, then the data center subdomain is us6. It’s also appended to your API key in the form key-dc; if your API key is 0123456789abcdef0123456789abcde-us6, then the data center subdomain is us6." from https://mailchimp.com/developer/marketing/docs/fundamentals/#:~:text=The%20root%20url%20for%20the,mailchimp.com%2F3.0%2F%20.

Looks like extracting should work!